src/ripple/protocol/tests/STAmount.test.cpp

```diff
@@ -404,24 +403,6 @@ class STAmount_test : public beast::unit_test::suite
     {
         testcase ("arithmetic");
 
```

Comment:
I'd feel a lot more comfortable about these changes if the `testRounding()` method in this file actually did something.  Right now it's pretty much all commented out.